static/elements/form-field-specs.js

```diff
@@ -160,6 +160,17 @@ export const ALL_FIELDS = {
         Comma separated list of full email addresses. These users will be allowed to edit this feature, but will not be listed as feature owners.`,
   },
 
+  'cc_recipients': {
+    type: 'input',
+    attrs: MULTI_EMAIL_FIELD_ATTRS,
+    required: false,
+    label: 'CC',
+    help_text: html`
+        Comma separated list of full email addresses. These users will be
+        carbon copied on any notification whenever a field of the feature is
```

Comment:
This makes it sound like the cc_recipients are listed on the CC: line of the emails that we send out.  That is not the case: we send each recipient of any kind an individual email that lists them on the To: line.

How about:
Comma separated list of full email addresses.  These users will be notified of any changes to the feature, but do not gain permission to edit.
?

Reply:
Will do!